bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -452,9 +446,9 @@ unsafe fn deallocate_output_data(ptr: *const u8, size: usize) {
 }
 
 /// Returns a pointer to a heap-allocated evmc_result.
-impl Into<*const ffi::evmc_result> for ExecutionResult {
-    fn into(self) -> *const ffi::evmc_result {
-        let mut result: ffi::evmc_result = self.into();
+impl From<ExecutionResult> for *const ffi::evmc_result {
```

Comment:
These changes were recommended by clippy and defining `From` gives both `From` and `Into` traits.